docs/ms-bio.md

```diff
@@ -0,0 +1,10 @@
+
+
+**Name:** Marc Sacks
+
+**Where you are from:** Belmont, 
```

Comment:
It looks like you might have forgotten to add a state here. :bow:  Is this Belmont, MA?